packages/cli/src/commands/devHandler.js

```diff
@@ -23,6 +24,71 @@ export const handler = async ({
 }) => {
   const rwjsPaths = getPaths()
 
+  // Starting values of ports from config (redwood.toml)
+  let apiPreferredPort = parseInt(getConfig().api.port)
+  let webPreferredPort = parseInt(getConfig().web.port)
+
+  // Assume we can have the ports we want
+  let apiAvailablePort = apiPreferredPort
+  let apiPortChangeNeeded = false
+  let webAvailablePort = webPreferredPort
+  let webPortChangeNeeded = false
+
+  // Check api port
+  if (side.includes('api')) {
+    apiAvailablePort = await getFreePort(apiPreferredPort)
+    if (apiAvailablePort === -1) {
+      errorTelemetry(
+        process.argv,
+        `Error could not determine a free port for the api server`
+      )
```

Comment:
I don't think we need to do any telemetry if we can't find a free port because it's not an error on redwood's part

```suggestion
```